### ISSUE TITLE
Adapt deepBGC to use GPUs

### ISSUE THREAD

**Issue by Darcy220606** [nf-core/funcscan#326]
### Description of feature

Adapt deepBGC to use GPU, at the moment it uses CPU and for large input datasets takes ages to run.

**Comment by jfy133** [comment]
I don't think we  will be able to do this for 1.2, because of having to re-building the container or something along those lines? @Darcy220606 thoughts?

**Comment by Darcy220606** [comment]
@jfy133 i was told once that we can assign a single process in the pipeline to use GPUs, in the configuration file  somewhere here 
```
process {
  withLabel:process_low {
    cpus = { check_max( 2 * task.attempt, 'cpus' ) }
    memory = { check_max( 14.GB * task.attempt, 'memory' ) }
    time = { check_max( 6.h * task.attempt, 'time' ) }
  }
```

**Comment by jfy133** [comment]
Hmm. Ok.. but I thought the tool has to be compiled for the particular GPU yo be used? I'm speaking completely naively though... Never had to dealt with it before myself

**Comment by jfy133** [comment]
Looking at deepBGC repo maybe I'm talking out of my butt...

I have a feeling though, if you are right it's about specifying this in a config file, that would imply you just send it to a node with a GPU? This would then be specific to particular clusters and thus not generally applicable within the code itself? 🤔

/Stream of consciousness
